go.mod

```diff
@@ -5,21 +5,24 @@ go 1.18
 require (
 	github.com/dsnet/golib/memfile v1.0.0
 	github.com/hashicorp/go-multierror v1.1.1
-	github.com/pion/dtls/v2 v2.2.8-0.20231201063746-dc751e3b2df9
+	github.com/pion/dtls/v2 v2.2.8-0.20240102042511-9ffd96c827fe
 	github.com/pion/transport/v3 v3.0.1
 	github.com/stretchr/testify v1.8.4
 	go.uber.org/atomic v1.11.0
-	golang.org/x/exp v0.0.0-20231127185646-65229373498e
+	golang.org/x/exp v0.0.0-20240103183307-be819d1f06fc
 	golang.org/x/net v0.19.0
-	golang.org/x/sync v0.5.0
+	golang.org/x/sync v0.6.0
 )
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/hashicorp/errwrap v1.1.0 // indirect
 	github.com/pion/logging v0.2.2 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
-	golang.org/x/crypto v0.16.0 // indirect
-	golang.org/x/sys v0.15.0 // indirect
+	golang.org/x/crypto v0.17.0 // indirect
+	golang.org/x/sys v0.16.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+// note: github.com/pion/dtls/v2/pkg/net package is not yet available in release branches
+exclude github.com/pion/dtls/v2 v2.2.9
```

Comment:
@hasheddan Hey there, do you know if pkg/net package will be added to release branches? It's available in the master branch, but not in the release branches (see v2.2.9 tag for example), so I have to exclude it here so the upgrade bot doesn't go for v2.2.9 and break compilation.